Xamarin.Forms.Core/TouchGestureRecognizer.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace Xamarin.Forms
+{
+	public class TouchGestureRecognizer : GestureRecognizer
+	{
+		public static readonly BindableProperty UseVisualStateManagerProperty =
```

Comment:
this shouldn't be optional. if a user define the State in his VSM, we should go to that state.

Reply:
done